onitu/api/__main__.py

```diff
@@ -30,16 +30,134 @@
 escalator = Escalator(session)
 logger = Logger("REST API")
 
+ONITU_READTHEDOCS = "https://onitu.readthedocs.org/en/latest/api.html"
+HOST = escalator.get(u'api:host') or 'localhost'
```

Comment:
It works since the value will always be there because it is loaded from the configuration by `start_setup` in  `onitu/__main__`. If not present in the configuration there will be "None" in the database.
I found it better to have the configuration "well defined" in the database, that is to say I preferred to always have the same values in the database for configuration, `port` and `host`.  If this is bad practice I can change that, but it does work as is ;)